### DISCUSSION TITLE
Issue with Non-Stationary Objects in Customized Scene

### DISCUSSION THREAD

**Discussion by ZhengtongXu** [StanfordVL/BEHAVIOR-1K#1021] [Q&A] [answered]
I've noticed that when I customize certain scenarios—such as initializing a bowl on a table—the bowl sometimes starts moving slowly in one direction even though the robot isn't applying any force on it. I suspect this issue might be due to the bowl not being perfectly positioned on the table at initialization. It seems like it starts slightly above the table and drops onto it, causing this initial motion, which results in the bowl not staying stationary. Do you have any ideas for resolving this? Perhaps adjusting the object's friction settings or similar parameters might help address this kind of issue? Thanks in advance

**Comment by hang-yin** [comment, marked as answer]
Hi @ZhengtongXu , that's a great question! This is a super common issue we would run into. 

One way to deal with this is once you load the scene, you can step a number of times to let the object, say the bowl, settle. You can try something like this: 

```
env = og.Environment(configs=cfg)

# Settle custom objects
for _ in range(100):
    og.sim.step()
```

The number of steps required here is arbitrary. But in general, I think for most objects, 100 steps would be good enough. I'd say even 20 steps can mostly get the job done. 

You might notice that after these steps, the joint state of the robot would also change a little due to external forces such as gravity. You can call `robot.reset()` to bring it back to its initial state after all objects are settled.

**Reply by ZhengtongXu** [reply]
Many thanks! Will try it
